Also confirmed on Windows `Ultralytics YOLOv8.2.7 🚀 Python-3.10.9 torch-2.1.1+cu121 CUDA:0 (NVIDIA GeForce RTX 3080, 12288MiB)`

```py
from ultralytics import YOLO

model = YOLO("yolov8n.pt")
result = model.train(data="coco8.yaml", epochs=16)
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "Q:\ML_dev\ultra_repo\ultralytics-1\ultralytics\engine\model.py", line 673, in train
    self.trainer.train()
  File "Q:\ML_dev\ultra_repo\ultralytics-1\ultralytics\engine\trainer.py", line 199, in train
    self._do_train(world_size)
  File "Q:\ML_dev\ultra_repo\ultralytics-1\ultralytics\engine\trainer.py", line 353, in _do_train
    for i, batch in pbar:
  File "Q:\ML_dev\ultra_repo\ultralytics\.ultra_docs\lib\site-packages\tqdm\std.py", line 1182, in __iter__
    for obj in iterable:
  File "Q:\ML_dev\ultra_repo\ultralytics-1\ultralytics\data\build.py", line 50, in __iter__
    yield next(self.iterator)
  File "Q:\ML_dev\ultra_repo\ultralytics\.ultra_docs\lib\site-packages\torch\utils\data\dataloader.py", line 630, in __next__
    data = self._next_data()
  File "Q:\ML_dev\ultra_repo\ultralytics\.ultra_docs\lib\site-packages\torch\utils\data\dataloader.py", line 1345, in _next_data
    return self._process_data(data)
  File "Q:\ML_dev\ultra_repo\ultralytics\.ultra_docs\lib\site-packages\torch\utils\data\dataloader.py", line 1371, in _process_data
    data.reraise()
  File "Q:\ML_dev\ultra_repo\ultralytics\.ultra_docs\lib\site-packages\torch\_utils.py", line 694, in reraise
    raise exception
AttributeError: Caught AttributeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "Q:\ML_dev\ultra_repo\ultralytics\.ultra_docs\lib\site-packages\torch\utils\data\_utils\worker.py", line 250, in _worker_loop
    init_fn(worker_id)
  File "Q:\ML_dev\ultra_repo\ultralytics-1\ultralytics\data\build.py", line 84, in seed_worker
    os.sched_setaffinity(0, range(NUM_THREADS))  # fix https://github.com/ultralytics/ultralytics/pull/11195
AttributeError: module 'os' has no attribute 'sched_setaffinity'
```

--- a/ultralytics/__init__.py
+++ b/ultralytics/__init__.py
@@ -1,6 +1,6 @@
 # Ultralytics YOLO 🚀, AGPL-3.0 license
 
-__version__ = "8.2.7"
+__version__ = "8.2.8"
 
 from ultralytics.data.explorer.explorer import Explorer
 from ultralytics.models import RTDETR, SAM, YOLO, YOLOWorld

--- a/ultralytics/data/build.py
+++ b/ultralytics/data/build.py
@@ -20,8 +20,9 @@ from ultralytics.data.loaders import (
     autocast_list,
 )
 from ultralytics.data.utils import IMG_FORMATS, VID_FORMATS
-from ultralytics.utils import NUM_THREADS, RANK, colorstr, LINUX
+from ultralytics.utils import LINUX, NUM_THREADS, RANK, colorstr
 from ultralytics.utils.checks import check_file
+
 from .dataset import GroundingDataset, YOLODataset, YOLOMultiModalDataset
 from .utils import PIN_MEMORY
 

--- a/ultralytics/data/build.py
+++ b/ultralytics/data/build.py
@@ -20,9 +20,8 @@ from ultralytics.data.loaders import (
     autocast_list,
 )
 from ultralytics.data.utils import IMG_FORMATS, VID_FORMATS
-from ultralytics.utils import NUM_THREADS, RANK, colorstr
+from ultralytics.utils import NUM_THREADS, RANK, colorstr, LINUX
 from ultralytics.utils.checks import check_file
-
 from .dataset import GroundingDataset, YOLODataset, YOLOMultiModalDataset
 from .utils import PIN_MEMORY
 
@@ -81,10 +80,8 @@ def seed_worker(worker_id):  # noqa
     worker_seed = torch.initial_seed() % 2**32
     np.random.seed(worker_seed)
     random.seed(worker_seed)
-    try:
+    if LINUX and hasattr(os, "sched_setaffinity"):
         os.sched_setaffinity(0, range(NUM_THREADS))  # fix https://github.com/ultralytics/ultralytics/pull/11195
-    except AttributeError:
-        pass  # <- CPU utilization may be suboptimal on Windows
 
 
 def build_yolo_dataset(cfg, img_path, batch, data, mode="train", rect=False, stride=32, multi_modal=False):

